packages/next/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -125,7 +138,11 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
     await span.traceChild('create-trace-assets').traceAsyncFn(async () => {
       const entryFilesMap = new Map<any, Set<string>>()
       const chunksToTrace = new Set<string>()
-      const isTraceable = (file: string) => !file.endsWith('.wasm')
+
+      const isTraceable = (file: string) =>
+        !NOT_TRACEABLE.some((suffix) => {
+          return file.endsWith(suffix)
+        })
```

Comment:
This is needed otherwise the build will error, since the image files are not traceable. See issue #39812.

In a seperate PR this plugin should be fixed.